app/Controllers/Api/V1/CategoriesController.php

```
@@ -28,52 +36,132 @@ public function __construct()
+
+        $allowedPayload = array_intersect_key($payload, $this->rules);
+
+        if (!$allowedPayload) {
+            return $this->failValidationErrors('Invalid payload');
+        }
+
+        if (!$this->validateData($allowedPayload, $this->rules)) {
+            return $this->failValidationErrors($this->validator->getErrors());
+        }
+
+        $allowedPayload['tenant_id'] = auth()->user()->tenant_id;
```

PHPStan (2.1.46)

[error] Access to an undefined property CodeIgniter\Shield\Entities\User::$tenant_id.
Learn more: https://phpstan.org/blog/solving-phpstan-access-to-undefined-property

(property.notFound)